.github/workflows/cmake_build.yml

```diff
@@ -7,11 +7,12 @@ name: CMake Build
 # Only execute on "git push" actions
 #
 on:
-  push:
+  workflow_dispatch:  
```

Comment:
We need to make sure, that this is not merged

Reply:
I make no comments on this file

Reply:
This changes when CI builds. Vitaly used it for testing